syntax = "proto3";

package model;

service ModelService {
  rpc GetStatus (Empty) returns (ModelStatus);
  rpc TrainModel (Empty) returns (TrainResponse);
}

message Empty {}

message ModelStatus {
  string status = 1;
}

message TrainResponse {
  string message = 1;
}
